### PR TITLE
update several docs

### PR DESCRIPTION
**Pull request by dguzenko**

### CODE REVIEW

File: 02.extensions/zoolanders/01.basics/02.installation/docs.md [hunk 1]
```diff
@@ -4,4 +4,13 @@ taxonomy:
...
+To install the ZOOlnaders extension, download the **zip** package and install it with **Joomla! Extension Manager** as any other plugin.
+
+## Installation with Joomla! Manager
+
+1. Download the package from the [Downloads](https://www.zoolanders.com/extensions/zoolanders)
+2. In your site, go to Joomla! Extension Manager and use the **Upload Package File** option to upload and install the downloaded
+
+>>> If you have issues with the installation try the alternative **Install from Directory** method uploading the unzipped package content into the `/tmp` folder of your site. Then input the path and click on the Install button.
+
+After installation make sure the extension is enabled, otherwise do so manually. Take into consideration Elements are hold under the same **ZL Elements** plugin for performance reasons.
```
Comment (miljan-aleksic) [inline 02.extensions/zoolanders/01.basics/02.installation/docs.md]: remove this line

File: 02.extensions/zoocart/01.basics/03.integration/docs.md [hunk 2]
```diff
@@ -28,11 +28,11 @@ Now that the elements are part of the product type we can assign them to a rende
 
 ## Subscriptions
 
-A **Subscription** is a product for which, after the Order has been completed, a subscription will be created and associated. You can choose the product type under the [ZOOcart element configuration](/zoocart/basics/configuration#zoocart).
+A **Subscription** is a product for which, after the Order has been completed, a subscription will be created and associated. You can choose the product type under the [ZOOcart element configuration](http://documentation.zoolanders.com/extensions/zoocart/basics/settings#zoocart).
```
Comment (miljan-aleksic) [inline 02.extensions/zoocart/01.basics/03.integration/docs.md]: make all urls relative

File: 02.extensions/zoolanders/01.basics/02.installation/docs.md (hunk 1 above)
Comment (dguzenko) [inline 02.extensions/zoolanders/01.basics/02.installation/docs.md]: done

File: 02.extensions/zoocart/01.basics/03.integration/docs.md (hunk 2 above)
Comment (dguzenko) [inline 02.extensions/zoocart/01.basics/03.integration/docs.md]: done

File: 02.extensions/zooaccess/01.basics/04.rules/docs.md [hunk 3]
```diff
@@ -74,7 +74,7 @@ This rule evaluates if the User is the Item Author or if it has modified it.
 
 ### ZOOcart Items
 
-This rule evaluates if the User has purchased the specified [ZOOcart](/zoocart) product.
+This rule evaluates if the User has purchased the specified [ZOOcart](/extensions/zoocart/basics) product.
```
Comment (miljan-aleksic) [inline 02.extensions/zooaccess/01.basics/04.rules/docs.md]: When linking to ext doc root remove the /basics

**Pull request merged**